editor/src/clj/editor/prefs.clj

```diff
@@ -22,6 +22,7 @@
     :any           anything goes, no validation is performed
     :boolean       a boolean value
     :string        a string
+    :password      a string, stored securely
```

Comment:
Calling this "securely" is a bit much, seeing as how you could easily figure out how to decrypt it from the source code. Maybe something like "not stored in clear text" or "stored encrypted" instead? 🤔